package.json

```diff
@@ -6,10 +6,10 @@
   },
   "filename": "brainstem",
   "standalone": "Brainstem",
-  "version": "0.4.13",
+  "version": "0.4.14-a",
   "author": {
-    "name": "Mavenlink",
-    "email": "dev@mavenlink.com"
+    "name": "Juan Carlos Medina",
```

Comment:
Will revert this soon. I was trying to get this repo transferred to the @mavenlink organization in the npmjs registry.